src/DotLiquid/Context.cs

```diff
@@ -698,7 +698,7 @@ private static object Liquidize(object obj)
             {
                 return liquidizableObj.ToLiquid();
             }
-            if (obj is string || obj is IEnumerable || obj is decimal || obj is DateTime || obj is DateTimeOffset || obj is TimeSpan || obj is Guid || obj is Enum
+            if (obj is IEnumerable /* string is IEnumerable */ || obj is decimal || obj is DateTime || obj is DateTimeOffset || obj is TimeSpan || obj is Guid || obj is Enum 

```

Comment:
```suggestion
            if (obj is IEnumerable /* string is IEnumerable */ || obj is decimal || obj is DateTime || obj is DateTimeOffset || obj is TimeSpan || obj is Guid || obj is Enum
```

(Trim trailing whitespace)